### ISSUE TITLE
sqlContext.createDataFrame throws error if you pass in array

### ISSUE THREAD

**Issue by doronrosenberg** [EclairJS/eclairjs-nashorn#44]
Standalone notebook code:

```
var jsc = new SparkContext("local[*]", "foo");
var sqlContext = new SQLContext(jsc);

var structField4 = DataTypes.createStructField("key", org.apache.spark.sql.types.DataTypes.IntegerType, true);
var structField5 = DataTypes.createStructField("value", org.apache.spark.sql.types.DataTypes.IntegerType, true);
var structType2 = DataTypes.createStructType([structField4,structField5]);
var dataFrame2 = sqlContext.createDataFrame([[1,1],[1,2],[2,1],[2,1],[2,3],[3,2],[3,3]], structType2);
```

Error is:

```
Name: javax.script.ScriptException
Message: ReferenceError: "sparkContext" is not defined in nashorn:mozilla_compat.js at line number 67
StackTrace: jdk.nashorn.api.scripting.NashornScriptEngine.throwAsScriptException(NashornScriptEngine.java:467)
jdk.nashorn.api.scripting.NashornScriptEngine.evalImpl(NashornScriptEngine.java:451)
jdk.nashorn.api.scripting.NashornScriptEngine.evalImpl(NashornScriptEngine.java:403)
jdk.nashorn.api.scripting.NashornScriptEngine.evalImpl(NashornScriptEngine.java:399)
jdk.nashorn.api.scripting.NashornScriptEngine.eval(NashornScriptEngine.java:155)
javax.script.AbstractScriptEngine.eval(AbstractScriptEngine.java:264)
org.eclairjs.nashorn.JavascriptInterpreter$$anonfun$2.apply(JavascriptInterpreter.scala:129)
org.eclairjs.nashorn.JavascriptInterpreter$$anonfun$2.apply(JavascriptInterpreter.scala:129)
scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
scala.concurrent.impl.ExecutionContextImpl$$anon$3.exec(ExecutionContextImpl.scala:107)
scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
```

**Comment by billreed63** [comment]
This code works for me, ideally you would use the JavaScript objects for the `DataTypes.createStructField` but the code will work with the Java objects as well. 
I pushed code to the master yesterday to createDataFrame from arrays, do you have the latest nashorn code?

**Comment by billreed63** [comment]
Fixed by f2cb80140f2e118b6739f1c85453db3972f28898
